docs/modules/ROOT/pages/migrate-from-cli.adoc

```diff
@@ -0,0 +1,242 @@
+= Migrate from OpenZeppelin CLI
+
+This guide will walk through migrating a project from the OpenZeppelin CLI to OpenZeppelin Upgrades Plugins for either Truffle or Buidler.
+
+include::partial$buidler-truffle-toggle.adoc[]
```

Comment:
We can include the snippet from the learn component directly. It may not work in the local preview though. This means the other file can be deleted from the PR.

```suggestion
include::learn::partial$buidler-truffle-toggle.adoc[]
```